getgather/mcp/amazon.py

```diff
@@ -63,9 +63,7 @@ async def search_product(keyword: str) -> dict[str, Any]:
 async def get_browsing_history() -> dict[str, Any]:
     """Get browsing history from amazon."""
 
-    async def get_browsing_history_action(
-        page: Page, browser_profile: BrowserProfile
-    ) -> dict[str, Any]:
+    async def get_browsing_history_action(page: Page, _) -> dict[str, Any]:
```

Comment:
just wonder why we need a second parameter? since action only send page as a callback argument